src/libraries/System.Private.CoreLib/src/System/IO/Strategies/AsyncWindowsFileStreamStrategy.ValueTaskSource.cs

```diff
@@ -18,9 +18,9 @@ internal sealed partial class AsyncWindowsFileStreamStrategy : WindowsFileStream
         private unsafe class ValueTaskSource : IValueTaskSource<int>, IValueTaskSource
```

Comment:
this type can be `sealed` now

```suggestion
        private sealed unsafe class ValueTaskSource : IValueTaskSource<int>, IValueTaskSource
```